src/sentry/tasks/members.py

```diff
@@ -63,3 +63,10 @@ def send_invite_request_notification_email(member_id):
         except Exception as e:
             logger = get_logger(name="sentry.mail")
             logger.exception(e)
+        analytics.record(
+            "invite_request.sent",
+            organization_id=om.organization.id,
+            user_id=member_id,
```

Comment:
@Neo-Zhixing So the `member_id` is not the same as the user id, you should use `om.user_id`

Reply:
turns out it should really be om.inviter.id